lib/text/ttml_text_parser.js

```diff
@@ -387,10 +399,12 @@ shaka.text.TtmlTextParser.parseCue_ = function(
  *
  * @param {!Element} regionElement
  * @param {!Array.<!Element>} styles
+ * @param {string} globalExtent
```

Comment:
Please add a comment with variant here or in the method description that explains what `globalExtent` is. Without that context, it is hard to know what it is for.